src/DependencyInjection/HWIOAuthExtension.php

```diff
@@ -172,3 +172,16 @@ private function createConnectIntegration(ContainerBuilder $container, array $co
         }
     }
 }
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
```

Comment:
This should be at top, same for that function declaration.

Reply:
done